Author: Jan Christoph Ebersbach <jceb@e-jc.de>
URL: no URL yet
Straight forward patch to focus the master window with a keybinding.

diff -r ad90e7fab364 focusmaster.c
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/focusmaster.c	Sat Mar 24 13:41:15 2012 +0100
@@ -0,0 +1,4 @@
+static void
+focusmaster(const Arg *arg) {
+	focus(selmon->clients);
+}
